k8s_scripts/LitmusUninstall.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-set -e 
+set -ex
```

Comment:
The directory structure can be:

`litmus/setup.sh` for setup
`litmus/cleanup.sh` for cleanup

Reply:
Sure